app/jobs/spree_mailchimp_ecommerce/create_user_job.rb

```diff
@@ -2,9 +2,12 @@ module SpreeMailchimpEcommerce
   class CreateUserJob < ApplicationJob
     def perform(user_id)
       user = Spree::User.find(user_id)
-      return unless user.mailchimp_user
+      return unless user
 
-      gibbon_store.customers.create(body: user.mailchimp_user)
+      mailchimp_user = user.mailchimp_user
```

Comment:
Where does the `mailchimp_<object>` get's created? Is there anything we can do if it's not there?

Reply:
It's added by this gem at `product_decorator`, so we don't need to worry about that.

Reply:
Is there a reason for it not to be present if the spree object is present?

Reply:
Sorry, I made a mistake.

https://github.com/JenniKayne/spree_mailchimp_ecommerce/blob/f8df2a758f378a75ca788b4ed7e8bf08d41293e3/app/models/spree/user_decorator.rb#L9